### PR TITLE
hip & rocm-opencl-runtime: fix for gcc 13

### PR DESCRIPTION
**Pull request by littlewu2508**

### CODE REVIEW

File: dev-libs/rocm-opencl-runtime/files/rocclr-5.3.3-gcc13.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
+From ca70cd7c3d63b1036e604116f3b269ca123e7904 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline dev-libs/rocm-opencl-runtime/files/rocclr-5.3.3-gcc13.patch:1]: Please always include a link to the upstream commit(s) and any relevant references (including Gentoo bugs) at the top of patches.

Thank you! I'll merge anyway in this case but going forward, it'd be a big help.

**Comment by thesamesam** [comment]
@littlewu2508 Is there a reason you're not in metadata.xml for these packages as a co-maintainer? (cc @candrews)

File: dev-libs/rocm-opencl-runtime/files/rocclr-5.3.3-gcc13.patch (hunk 1 above)
Comment (littlewu2508) [reply dev-libs/rocm-opencl-runtime/files/rocclr-5.3.3-gcc13.patch:1]: I'll add the link to the upstream commit

**Comment by littlewu2508** [comment]
> @littlewu2508 Is there a reason you're not in metadata.xml for these packages as a co-maintainer? (cc @candrews)

History: the rocm stack is previously maintained by @justxi (proxy) and @candrews, mainly for the low-level components (runtimes). @justxi also maintains an overlay at https://github.com/justxi/rocm with the full stack, while the high level libraries are highly unstable.

I entered the "Gentoo rocm team" half way to get hip and ROCm libraries working, when low-level components are already in ::gentoo with basic quality. My contribution mainly focus on hip and high level math libraries like rocBLAS, where I'm the proxy maintainer.

Then at some time @candrews "retired" (there's no rocm team actually, so there's no official retirement -- I mean he's been inactive in ROCm ebuilds. I remember he was said to drop the AMD card, thus loosing the test environment, but I cannot find reference)

As of @justxi I don't know what he thinks, maybe for he's satisfied for the status quo at ::gentoo, so he just did some development at his own overlay. I have been taking over the maintenance then, mainly bumping versions, fixing bugs and enable testings to enhance quality.

So, it's true that I should put my name in the maintainer list now.

File: dev-libs/rocm-opencl-runtime/files/rocclr-5.3.3-gcc13.patch (hunk 1 above)
Comment (littlewu2508) [reply dev-libs/rocm-opencl-runtime/files/rocclr-5.3.3-gcc13.patch:1]: Oh I see you already closed it. I'll put the link and my name in maintainer list in another PR

**Pull request closed without merging**